app/src/main/res/values/strings.xml

```diff
@@ -141,6 +141,14 @@
     <string name="about_contributors_description">Those who contribute to creation.</string>
     <string name="about_version_title">Version</string>
     <string name="about_version_description">%1$s</string>
+    <string name="about_what_is_prefix">What is " </string>
```

Comment:
Please add `translatable="false" ` if you think that translation is not needed. 🙏

Reply:
I added in https://github.com/DroidKaigi/conference-app-2018/pull/572/commits/38c827348f6d5e7c496f3c0aff10942f9d37f15d